src/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_EXTENSIONS ON)
```

Comment:
There *might* be some bad interactions between this and the `-std=` flag added by Corrade with the `CORRADE_CXX_STANDARD` property. I'll need to check.

Reply:
I didn't notice anything weird on my end so far. Both with the bootstrap project and my main project, but maybe I'm just lucky.

Reply:
For now I'll change this to `CORRADE_CXX_STANDARD` like everywhere else, which kinda works around the need to test anything :D